tests/test_geoprocessing.py

```diff
@@ -201,6 +201,32 @@ def test_reclassify_raster_empty_value_map(self):
                 (raster_path, 1), empty_value_map, target_path,
                 gdal.GDT_Float32, target_nodata, values_required=False)
 
+    def test_reclassify_raster_int_to_float(self):
+        """PGP.geoprocessing: test reclassify to float types."""

```

Comment:
This seems like a good test for basic functionality of `reclassify_raster`, regardless of the previous comment.